example-playbooks/manage_users/create_user.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Create user {{ username }}
+  block:
+  - set_fact:
+      superuser_str: "{% if superuser|bool %}SUPERUSER{% else %}NOSUPERUSER{% endif %}"
+  - shell: |
+      cqlsh {{ broadcast_address }} -u {{ admin_username }} -p {{ admin_password }} -e "CREATE USER IF NOT EXISTS {{ username }} WITH PASSWORD '{{ password }}' {{ superuser_str }};"
```

Comment:
`broadcast_address` is not mentioned in the patch description.

Please, duplicate the usage and expected parameters description in the file as well.

An usage example would be welcome as well.

Reply:
> broadcast_address is not mentioned in the patch description.

I removed `broadcast_address` from the parameters and now I'm getting it directly from scylla.yaml

> Please, duplicate the usage and expected parameters description in the file as well.

Done

> An usage example would be welcome as well.

Done

Reply:
Done